www/convertUtils.js

```diff
@@ -28,10 +28,19 @@ module.exports = {
     */
     toCordovaFormat: function (contact) {
         var value = contact.birthday;
-        try {
-          contact.birthday = new Date(parseFloat(value));
-        } catch (exception){
-          console.log("Cordova Contact toCordovaFormat error: exception creating date.");
+        if (value !== null) {
+            try {
+              contact.birthday = new Date(parseFloat(value));
+              
+              //we might get 'Invalid Date' which does not throw an error
+              //and is an instance of Date.
+              if (isNaN(contact.birthday.getTime())) {
+                contact.birthday = null;
```

Comment:
code looks good to me, could you please add a comment about the fact that when birthday is null, native side(iOS, Android,etc...) will return an 'Invalid Date' ? That will make it easier for others to grasp the reason behind this.
After that, we'll be good to go.